packages/terra-form-field/src/Field.scss

```diff
@@ -0,0 +1,78 @@
+:local {
+  .field {
+    border: 0;
+    margin-bottom: var(--terra-form-field-margin-bottom, 1.428em);
+    max-width: 1020px;
+    padding: 0;
+  }
+
+  .field-inline {
+    display: inline-block;
+    margin-right: var(--terra-form-field-inline-margin-right, 0.7143em);
+    vertical-align: top;
+  }
+
+  .label-group {
+    font-family: var(--terra-form-field-label-font-family, inherit);
+    font-size: var(--terra-form-field-label-font-size, 1rem);
+    line-height: 1.29;
+    margin-bottom: var(--terra-form-field-label-margin-bottom, 0.357em);
+    text-align: left;
+  }
+
+  .label-group-hidden {
+    display: none;
+  }
+
+  .label {
+    color: var(--terra-form-field-label-color, rgba(0, 0, 0, 0.8));
+    font-weight: var(--terra-form-field-label-font-weight, bold);
+  }
+
+  .error-icon,
+  .error-icon-hidden {
+    display: inline-block;
+    padding-right: var(--terra-form-field-label-error-icon-padding-right, 0.357em);
+  }
+
+  .error-icon-hidden {
+    visibility: hidden;
+  }
+
+  .required,
+  .required-hidden {
+    color: var(--terra-form-field-label-required-color , #ff232a);
+    display: inline-block;
+    padding-right: var(--terra-form-field-label-required-padding-right, 0.357em);
+  }
+
+  .required-hidden {
+    visibility: hidden;
+  }
+
+  .optional {
+    display: inline-block;
+    font-style: var(--terra-form-field-label-optional-font-style, italic);
+    padding-left: var(--terra-form-field-label-optional-padding-left, 0.357em);
+  }
+
+  .help-text {
+    color: var(--terra-form-field-help-text-color, rgba(0, 0, 0, 0.5));
+    font-family: var(--terra-form-field-help-text-font-family, inherit);
```

Comment:
Should this be a themeable variable? I think the original request wanted this to use the value of --terra-base-font-family, which inherit would take care of?

Reply:
Though maybe this is an all encompassing theme for base then.

Reply:
@bjankord Did I miss-interpret your request?

Reply:
font-family can be removed from this instance. The help-text font family should be the same font family as what is set on the html element and the help-text div will inherit that font through the cascade without us needing to set any styles.